etcdserver/membership/cluster.go

```diff
@@ -307,21 +307,21 @@ func (c *RaftCluster) RemoveMember(id types.ID) {
 	c.removed[id] = true
 }
 
-func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) bool {
+func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) {
```

Comment:
Or if not `SetVersion` because of path mismatch, what about `f` in https://github.com/coreos/etcd/pull/5005/files#diff-971f6af906b31cc86efd71aff56aa96bR1106 ?